### PR TITLE
Misc. formatting fixes

### PR DESCRIPTION
**Pull request by hazelnot**
- Added padding to navbar
- Removed dash from reusable
- Added page formatting as workaround for white strip appearing on the right of the page

This is my first time working on someone else's code, so it's probably not the best pr ever but I hope it helps

### CODE REVIEW

File: themes/hugo-theme-qaul/static/css/qaul.css [hunk 1]
```diff
@@ -98,6 +98,7 @@ nav li {
 /* tab navigation */
 header {
 	background-color: var(--c-blue-bg);
+	padding-top:1rem;
```
Comment (spacekookie) [inline themes/hugo-theme-qaul/static/css/qaul.css]: Maybe fix the formatting there

**Review by spacekookie — COMMENTED** [review]
Hey there, thanks for the PR :heart: 

So, am I correct to assume that this PR is supposed to always make the header full-height of the screen, no matter of the width?

While that does look cool, it does if course mean that there is a lot more scrolling to be done before getting to any content. But not sure if the alternative of making the first header frame _not_ full height, even on desktop, is gonna look that great.

@MathJud, any thoughts? (also I know he's not gonna look at the website for a while because it's been driving him mad :wink:)

**Comment by hazelnot** [comment]
> So, am I correct to assume that this PR is supposed to always make the header full-height of the screen, no matter of the width?

Unless it's a phone screen, actually

File: themes/hugo-theme-qaul/static/css/qaul.css (hunk 1 above)
Comment (hazelnot) [reply themes/hugo-theme-qaul/static/css/qaul.css]: Done!

**Comment by MathJud** [comment]
Thanks for this PR! I think it looks really nice!

I detected the following problems:

* there is no right padding in the lower title, in some scalings the title text runs until under the scrollbar.
* in some small scalings the title overreaches the lower end of the blue field (tested with firefox)

I have the following suggestions:

* Some of the scalings look weired
  * the big scaling has the lower title on one line which does
  * The background logo in the small height screens could be bigger and more in the center maybe ..
* It would be nice to have an arrow at the bottom of the page for that people know they should scroll down.

**Comment by hazelnot** [comment]
I tried to split the subtitle onto multiple lines, but I couldn't figure out how to add a line break in the format Hugo uses.

Also, the logo was originally even more offset, and I thought it was intentional?

**Comment by MathJud** [comment]
> I tried to split the subtitle onto multiple lines, but I couldn't figure out how to add a line break in the format Hugo uses.

I Wouldn't use line breaks but padding left and right.

> Also, the logo was originally even more offset, and I thought it was intentional?

Yes it was/is intentionally. But the layout / font-items remained at the same location too.
Now the font items are visually moving down & the logo is moving up.

**Pull request merged**